### PR TITLE
Sv 493 epao withdraw from a standard version

### PR DESCRIPTION
**Pull request by StuartReeve102**

### CODE REVIEW

File: src/SFA.DAS.AssessorService.Web/Controllers/ApplyForWithdrawalController.cs [hunk 1]
```diff
@@ -114,10 +119,17 @@ public async Task<IActionResult> ChooseStandardForWithdrawal()
                 ? result
                 : (int?)null;
 
+            // remove any standards where there is an existing withdrawal application
```
Comment (alandburns) [inline src/SFA.DAS.AssessorService.Web/Controllers/ApplyForWithdrawalController.cs:122]: You should not remove from the list any standards, whether or not they have an in progress withdrawal / withdrawal by version. Instead all should be listed and if the user selects one that has an in progress withdraw they should be informed with a Warning message. This maay require input from the UX team.

File: src/SFA.DAS.AssessorService.Application.UnitTests/Handlers/Applications/OrganisationStandardVersionOptInHandlerTests/WhenHandlingOrganisationStandardVersionOptInRequest.cs [hunk 2]
```diff
@@ -1,14 +1,18 @@
...
+﻿using AutoFixture.NUnit3;
+using FluentAssertions;
 using MediatR;
 using Microsoft.Extensions.Logging;
 using Moq;
 using Newtonsoft.Json;
 using NUnit.Framework;
 using SFA.DAS.AssessorService.Api.Types.Consts;
 using SFA.DAS.AssessorService.Api.Types.Models;
+using SFA.DAS.AssessorService.Api.Types.Models.AO;
```
Comment (CodeWolf86) [inline src/SFA.DAS.AssessorService.Application.UnitTests/Handlers/Applications/OrganisationStandardVersionOptInHandlerTests/WhenHandlingOrganisationStandardVersionOptInRequest.cs:10]: Can these usings be removed?

File: src/SFA.DAS.AssessorService.Application.Api/Controllers/Validations/WithdrawalDateValidationController.cs [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+        public async Task<ActionResult<ApiValidationResult>> ValidateWithdrawalDate(Guid id, string q)
+        {
+            try
+            {
+                var application = await _mediator.Send(new GetApplicationRequest(id));
+                var appData = await _qnaApiClient.GetApplicationData(application.ApplicationId);
+
+                var dateComponents = q.Split(",");
+                var enteredDate = new DateTime(int.Parse(dateComponents[2]), int.Parse(dateComponents[1]), int.Parse(dateComponents[0]));
+                if (enteredDate < appData.EarliestDateOfWithdrawal.Date)
```
Comment (CodeWolf86) [inline src/SFA.DAS.AssessorService.Application.Api/Controllers/Validations/WithdrawalDateValidationController.cs:38]: The earliest date of withdrawal that's stored with the application data. How is that calculated? Is that something we can get here a different way? It feels odd to have to call the QnA Api and setup all the config just to get that one thing if we can replicate the calculation. If it's not possible, happy with what's here, just wanted to understand the logic under the hood QnA side

**Review by CodeWolf86 — CHANGES_REQUESTED** [review]
Minor Style issue on one of the pages: 
image

File: src/SFA.DAS.AssessorService.Application.UnitTests/Handlers/Applications/OrganisationStandardVersionOptInHandlerTests/WhenHandlingOrganisationStandardVersionOptInRequest.cs (hunk 2 above)
Comment (StuartReeve102) [reply src/SFA.DAS.AssessorService.Application.UnitTests/Handlers/Applications/OrganisationStandardVersionOptInHandlerTests/WhenHandlingOrganisationStandardVersionOptInRequest.cs:10]: Fixed

File: src/SFA.DAS.AssessorService.Application.Api/Controllers/Validations/WithdrawalDateValidationController.cs (hunk 3 above)
Comment (StuartReeve102) [reply src/SFA.DAS.AssessorService.Application.Api/Controllers/Validations/WithdrawalDateValidationController.cs:38]: Removed the qna api call, now using the method  that calculates the earliestDateOfWithdrawal

File: src/SFA.DAS.AssessorService.Web/Views/Application/Submitted.cshtml [hunk 4]
```diff
@@ -24,7 +24,7 @@
 
                     <div class="govuk-panel govuk-panel--confirmation">
                         <h1 class="govuk-panel__title">
-                            Withdrawal notification submitted
+                            Withdrawal request submitted
```
Comment (ben1stone) [inline src/SFA.DAS.AssessorService.Web/Views/Application/Submitted.cshtml:27]: Content for notification > request also needs updating in the What happens next paragraph

**Review by ben1stone — CHANGES_REQUESTED** [review]

File: src/SFA.DAS.AssessorService.Web/Controllers/ApplyForWithdrawalController.cs (hunk 1 above)
Comment (StuartReeve102) [reply src/SFA.DAS.AssessorService.Web/Controllers/ApplyForWithdrawalController.cs]: This is no longer applicable with the UI changes

File: src/SFA.DAS.AssessorService.Web/Views/Application/Submitted.cshtml (hunk 4 above)
Comment (StuartReeve102) [reply src/SFA.DAS.AssessorService.Web/Views/Application/Submitted.cshtml:27]: Fixed

**Review by CodeWolf86 — APPROVED** [review]

**Pull request merged**